src/views/InvolvementProfile/components/ContactListItem/index.js

```diff
@@ -43,7 +43,11 @@ const ContactListItem = ({ contact }) => {
         </Avatar>
       </ListItemAvatar>
       <Link href={`/profile/${contact.Email.split('@')[0]}`}>
-        <ListItemText primary={`${contact.FirstName} ${contact.LastName}`} />
+        <ListItemText
+          primary={`${contact.FirstName} ${contact.LastName} ${
+            contact.Description?.trim() ? '- ' + contact.Description : ''
+          }`}
```

Comment:
Do you think it would look nicer as secondary list item text?
```suggestion
          primary={`${contact.FirstName} ${contact.LastName}`}
          secondary={contact?.Description}
```